icu4c/source/common/servnotf.cpp

```diff
@@ -59,7 +59,7 @@ ICUNotifier::addListener(const EventListener* l, UErrorCode& status)
                 }
```

Comment:
Ah, GitHub won't let comment on the line above... but I wonder if it would be worth adding a `LocalPointer<UVector>` on line 52 for the member `listeners`, in case OOM occurs.

Reply:
Fixed. Sorry I missed that one.